services/ui_backend_service/data/cache/get_parameters_action.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+from .get_data_action import GetData
+
+from metaflow import Step
+
+
+class GetParameters(GetData):
+    @classmethod
+    def format_request(cls, pathspecs: List[str], invalidate_cache=False):
```

Comment:
maybe naming could be specific, as this will only work for **run** pathspecs due to the appending of `_parameters` to get a step?

Reply:
Added more documentation on input parameters